libs/core/src/lib/facets/facet/facet.component.html

```diff
@@ -3,7 +3,9 @@
 </ng-container>
 
 <ng-container *ngIf="type !== 'image'">
-    <h1 *ngIf="facetTitle" fd-title [headerSize]="5" class="fd-margin-bottom--sm" [id]="titleId">{{ facetTitle }}</h1>
+    <span *ngIf="facetTitle" fd-title [headerSize]="5" class="fd-margin-bottom--sm" [id]="titleId">{{
+        facetTitle
+    }}</span>
```

Comment:
That's actually not a good solution. In my opinion, what Nikolay suggested is the best option:

```
For 2 you need to take the heading level hierarchy and nesting into account. If you do not know which heading level is expected according to a floorplan and the usage/nesting of the control is not clear then you need to expose an API, so that the app devs can adjust this accordingly.
```

The facet title should be something the developers can specify based on their headings order in the application.